autogpt_platform/backend/backend/executor/manager.py

```diff
@@ -9,41 +8,32 @@
 from concurrent.futures import Future, ProcessPoolExecutor
 from contextlib import contextmanager
 from multiprocessing.pool import AsyncResult, Pool
-from typing import TYPE_CHECKING, Any, Coroutine, Generator, TypeVar, cast
```

Comment:
Changes here: Replacing direct model call to db_client call. 
And remove event loop that is no longer used for doing prisma queries.